src/api-request.js

```diff
@@ -1,42 +1,31 @@
 let apiRequest = {
-  userInfo: getUserData(),
-  sleepInfo: getSleepData(),
-  hydrationInfo: getHydrationData(),
-  activityInfo: getActivityData()
-};
-function getUserData() {
-  return fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData')
+getUserData() {
```

Comment:
Oh I see. create the methods within the object literal instead of invoking them inside of it.